src/cupbearer/scripts/conf/train_classifier_conf.py

```diff
@@ -64,6 +35,22 @@ def __post_init__(self):
         if isinstance(self.model, (MLP, CNN)):
             self.model.output_dim = self.num_classes
 
+        # For datasets that are not necessarily deterministic based only on
+        # arguments, this is where validation sets are set to follow train_data
+        if hasattr(self.train_data, "backdoor"):
+            for name, val_config in self.val_data.items():
+                # WanetBackdoor
+                try:
+                    str_factor = (
+                        val_config.backdoor.warping_strength
+                        / self.train_data.backdoor.warping_strength
+                    )
+                    val_config.backdoor.control_grid = (
```

Comment:
Similar comment here to the one above: overriding the property with a value feels a bit strange (but probably doesn't actually hurt)

Reply:
This should be using the setter no? But, I'll make sure to double check this as well.